packages/connect/src/events/blockchain.ts

```diff
@@ -18,6 +19,11 @@ export interface BlockchainInfo extends ServerInfo {
     };
 }
 
+export interface BlockchainReconnecting {
+    coin: CoinInfo;
+    time: number;
```

Comment:
wouldn't `attempt` field be useful? if not used anywhere or planned anywhere, just "kaslat na to".

Reply:
Dunno, it was currently relevant only for calculating the delay until the next reconnection. If it'll be needed in the future, it's pretty easy to add it I think.

Reply:
sure